docs/api-constructor.md

```diff
@@ -35,6 +35,7 @@ Implements the [stream.Duplex][1] class.
         -   `options.raw.width` **[number][8]?** integral number of pixels wide.
         -   `options.raw.height` **[number][8]?** integral number of pixels high.
         -   `options.raw.channels` **[number][8]?** integral number of channels, between 1 and 4.
+        -   `options.raw.premultiplied` **[boolean][7]?** specifies that the raw input has already been premultiplied, set to `true` to avoid sharp premultiplying the image. (optional, default `false`)
```

Comment:
Please can you add this to the JSDocs at https://github.com/lovell/sharp/blob/master/lib/constructor.js#L141 then re-generate the markdown using `npm run docs-build`